core/src/main/java/org/elasticsearch/index/query/SimpleQueryParser.java

```diff
@@ -202,51 +203,102 @@ private Query newPossiblyAnalyzedQuery(String field, String termStr) {
             return new PrefixQuery(new Term(field, termStr));
         }
     }
-
     /**
      * Class encapsulating the settings for the SimpleQueryString query, with
      * their default values
      */
-    public static class Settings {
-        private Locale locale = Locale.ROOT;
-        private boolean lowercaseExpandedTerms = true;
-        private boolean lenient = false;
-        private boolean analyzeWildcard = false;
+    static class Settings {
+        /** Locale to use for parsing. */
+        private Locale locale = SimpleQueryStringBuilder.DEFAULT_LOCALE;
+        /** Specifies whether parsed terms should be lowercased. */
+        private Boolean lowercaseExpandedTerms = SimpleQueryStringBuilder.DEFAULT_LOWERCASE_EXPANDED_TERMS;
+        /** Specifies whether lenient query parsing should be used. */
+        private Boolean lenient = SimpleQueryStringBuilder.DEFAULT_LENIENT;
+        /** Specifies whether wildcards should be analyzed. */
+        private Boolean analyzeWildcard = SimpleQueryStringBuilder.DEFAULT_ANALYZE_WILDCARD;
```

Comment:
Than I need to re-add the null-check there:

https://github.com/elastic/elasticsearch/pull/11274#discussion-diff-32930795